website/content/en/docs/getting-started/users.md

```diff
@@ -103,8 +91,9 @@ To deploy ClusterLink on another cluster, please repeat steps 2-3 in the console
     EOF
     ```
 
-    The instance CRD will create the ClusterLink gateway components in the `clusterlink-system` namespace.
-    For more details and information about the ClusterLink instance CRD, refer to the [operator documentation](https://github.com/clusterlink-net/clusterlink/blob/main/design-proposals/project-deployment.md#clusterlink-crd).
+    After the operator is installed, you can deploy ClusterLink by applying a ClusterLink object as shown above.
+    The ClusterlinkOperator will create the ClusterLink gateway components in the `clusterlink-system` namespace.
```

Comment:
I don't expect the term `ClusterLinkOperator` to be meaningful to the user. Maybe just `operator` or `it`?